From 3fb579e118106932241e6f8193fb2c0f28e5ca13 Mon Sep 17 00:00:00 2001
From: yuhao_zhang <yuhao.zhang@huawei.com>
Date: Mon, 23 Feb 2026 13:48:55 +0800
Subject: [PATCH 12/18] feat: assign versions in the spec

Signed-off-by: yuhao_zhang <yuhao.zhang@huawei.com>
---
 src/libobmm/CMakeLists.txt | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/src/libobmm/CMakeLists.txt b/src/libobmm/CMakeLists.txt
index 22b8c84..3932217 100644
--- a/src/libobmm/CMakeLists.txt
+++ b/src/libobmm/CMakeLists.txt
@@ -10,8 +10,8 @@ target_include_directories(OBMM_SO PRIVATE ${CMAKE_CURRENT_LIST_DIR})
 
 set(OBMM_NAME obmm)
 set_target_properties(OBMM_SO PROPERTIES OUTPUT_NAME ${OBMM_NAME}
-    VERSION 1.0.1
-	SOVERSION 1)
+	VERSION ${OBMM_VERSION}
+	SOVERSION ${OBMM_SOVERSION})
 
 target_compile_options(OBMM_SO PRIVATE -Wall -Wextra -Wfloat-equal -fno-common -std=gnu99 -fvisibility=hidden
 	-Wuninitialized -Wno-error -Wno-error=format -Wundef -Wunused -Wdate-time -Wshadow -Wvla
@@ -32,4 +32,5 @@ else()
 	target_link_options(OBMM_SO PRIVATE -s)
 endif()
 
-install(TARGETS OBMM_SO LIBRARY DESTINATION lib OPTIONAL)
+install(TARGETS OBMM_SO LIBRARY DESTINATION lib64 OPTIONAL)
+install(FILES libobmm.h DESTINATION include)
\ No newline at end of file
-- 
2.43.0

